tasks/apm-inject-install.yml

```diff
@@ -0,0 +1,39 @@
+---
+- name: Include APM host injection Debian install tasks
+  include_tasks: pkg-debian/install-apm-inject-latest.yml
+  when: not ansible_check_mode and ansible_facts.os_family == "Debian" and datadog_apm_instrumentation_enabled != ""
```

Comment:
This whole file is only included when `datadog_apm_instrumentation_enabled != ""`, so let's not check this specific condition again throughout this file.

Reply:
removed check